<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoStatisticsWidget</class>
 <widget class="QWidget" name="VideoStatisticsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>FPS:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="frameRate">
          <property name="baseSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Bitrate:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="bitRate">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11" stretch="0,0">
        <item>
         <widget class="QLabel" name="lBitrate">
          <property name="text">
           <string>Resolution:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="resolution">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Quantization:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="quantization">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Frame types:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="frameTypes">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Frames dropped:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="framesDropped">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12" stretch="0,0">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Delay:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="delay">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14" stretch="0,0">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Frames count:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="framesCount">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Bytes read:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="streamSize">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Frame size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="frameSize">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Time elapsed:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="timeElapsed">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
